include/geom/face_tri3.h

```diff
@@ -175,6 +175,27 @@ class Tri3 : public Tri
 
 };
 
+class TriShell3 : public Tri3
```

Comment:
Same comment about separate C/h file for new class, plus Doxygen-style documentation.